include/NovelRT/DebugService.h

```diff
@@ -2,25 +2,23 @@
 
 #ifndef NOVELRT_NOVELDEBUGSERVICE_H
 #define NOVELRT_NOVELDEBUGSERVICE_H
-
-#include "NovelTextRect.h"
-#include <glad/glad.h>
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
 
 namespace NovelRT {
 
-  typedef class NovelRunner NovelRunner;
-
-  class NovelDebugService {
+  class DebugService {
 
   private:
     NovelRunner* _runner;
-    NovelTextRect* _fpsCounter;
+    std::unique_ptr<Graphics::TextRect> _fpsCounter;
```

Comment:
just a note.

eventually this should *probably* be something added to the scene graph and enabled/disabled by the debug service

Reply:
We can subscribe to the scene construction. I'll do that now. Forgot. Oops lol.